application/F3DStarter.cxx

```diff
@@ -1512,7 +1512,7 @@ int F3DStarter::AddFile(const fs::path& path, bool quiet)
       bool found = false;
       std::vector<std::vector<fs::path>>::iterator it;
       for (it = this->Internals->FilesGroups.begin(); it != this->Internals->FilesGroups.end();
-           it++)
+        it++)
```

Comment:
for some reason your local clang-format is slightly different than our CI, lets not worry about it for now.